cli/src/node.rs

```diff
@@ -0,0 +1,46 @@
+// Copyright 2024 Hoku Contributors
```

Comment:
unneeded file?

Reply:
This was not supposed to be in the PR, my bad